index.js

```diff
@@ -73,7 +73,7 @@ if (!isProduction) {
 
 // production error handler
 // no stacktraces leaked to user
-app.use((err, req, res) => {
+app.use((err, req, res, next) => {
```

Comment:
Please address this hound isssue

Reply:
I have done so

Reply:
However, according to express documentation on error handling, four arguments must be passed into that middleware for it to be recognized as an error handler, even if the last argument, `next` will not be used. 
![image](https://user-images.githubusercontent.com/32905583/43954299-e2832cee-9c93-11e8-8aff-7c9e28c7fa2a.png)
What I did now to fix the hound issue was just to call the `next` object. But it is actually not needed since there is no other route handler after error handler presently. This affects the test coverage by a small amount too (from 90 to 89).

Besides, this is actually a bug fix whose PR is opened here https://github.com/andela/fargo-ah/pull/27. You may consider merging it soonest so it can fix the 'error handling bug' before other upcoming branches are merged.

https://expressjs.com/en/guide/using-middleware.html#middleware.error-handling